docs/concepts/assets/accounts.md

```diff
@@ -89,14 +93,41 @@ The revenue from other fees go into fee-specific accounts:
 [Fees](../trading-on-vega/fees-rewards.md)
 :::
 
-### Reward accounts 
-Trading rewards are funded by community members that want to incentivise certain types of market activities. 
+### Trading reward accounts 
+Trading rewards are funded by community members that want to incentivise certain types of market activities.
 
 Reward accounts can exist for each combination of reward metric, asset, and market, and they come to exist when they are funded by reward account transfers, which occur at the end of each epoch. The amount of assets entering each reward account depends on the amount transferred and the reward transfer strategy that the funder(s) defined when they set up the rewards transfer.
 
+There is also a global rewards account, which is used for validator rewards on top of infrastructure fee revenue.
+
+| Reward account name                               | Account purpose |
+|---------------------------------------------------|-----------------|
+| Vested rewards      | Holds earned rewards that have passed the vesting period and can be redeemed |
+| Vesting rewards | Holds earned rewards that are still within the vesting period |
+| Maker fees paid reward | Holds funds allocated to reward aggressive side of a trade based on fees paid |
+| Maker fees received reward | Holds funds allocated to reward passive side of a trade based on fees earned |
+| LP received fees reward     | Holds funds allocated to reward liquidity providers based on fees received for trades hit |
+| Average position reward | Holds funds allocated to reward the largest traders by position size |
+| Relative return reward | Holds funds allocated to reward most profitable traders |
+| Return volatility reward | Holds funds allocated to reward traders with the most consistent returns |
+| Market proposers reward | Holds funds allocated to reward creating markets that attract good trading volume |
+
 :::note Read more
 [Trading rewards](../trading-on-vega/fees-rewards.md#trading-rewards)
 :::
 
-<!--### Network treasury accounts
-The network treasury is made up of accounts, one per asset (if funded), that are contributed to by participants in the network. They can be funded through deposits or direct transfers from other protocol-governed accounts. In the future, funds in the network treasury can be used for rewards, grants, or other uses defined by tokenholder governance. -->
+### Validator reward accounts
+Validator reward accounts are used to distribute rewards, if they're funded, to consensus and standby validators based on their ranking score. These accounts can be funded through [deposits](./deposits-withdrawals.md) or [transfers](./transfers.md).
+
+The assets in the validator reward accounts can only otherwise move if a [governance-initiated transfer](./transfers.md#governance-initiated-transfers) is enacted, meaning a governance proposal to move assets has passed the vote.
```

Comment:
do you mean global reward account in this section?

Reply:
I meant the reward pool that funds the validator rewards, but I will rephrase to make it more precise.